source/game/field/obj/ObjectColMgr.hh

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "game/field/CourseColMgr.hh"
+#include "game/field/KColData.hh"
+
+#include <egg/math/BoundBox.hh>
+#include <egg/math/Matrix.hh>
+
+namespace Field {
+
+class ObjectColMgr {
+public:
+    static void *LoadFile(const char *filename);
+    ObjectColMgr(const char *filename);
+    ~ObjectColMgr();
+
+    void setMtx(EGG::Matrix34f &val);
+    void setKCLScale(f32 val);
+
+    EGG::Vector3f getKclBboxLowWorld();
+    EGG::Vector3f getKclBboxHighWorld();
```

Comment:
```suggestion
    const EGG::Vector3f &getKclBboxHighWorld() const;
```

Reply:
This isn't a getter, it calculates the return value.